src/js/angular/core/directives/queryeditor/templates/query-editor.html

```diff
@@ -211,7 +211,7 @@ <h6 class="alert alert-warning no-icon additional" ng-show="!queryIsRunning && !
                     <h6>Error<span ng-show="yasr.results.getException().status > 0"> {{yasr.results.getException().status}}: {{yasr.results.getException().statusText}}</span></h6>
                 </div>
 
```

Comment:
We try to avoid using inline css styles [see point 13 here](https://docs.google.com/document/d/1SwGD0X-797-06QM_PPYCM3vZO5iFDJIaCFV8deGTZyw/edit#heading=h.z7jjpwx6gyul)
It would be better if you can find a proper stylesheet where to put these styles.